server/src/common/server.ts

```diff
@@ -814,8 +815,8 @@ export class _LanguagesImpl implements Remote, _Languages {
 	}
 }
 
-export type Languages = _Languages & CallHierarchy & SemanticTokensFeatureShape & LinkedEditingRangeFeatureShape & TypeHierarchyFeatureShape & InlineValueFeatureShape & InlayHintFeatureShape & DiagnosticFeatureShape & MonikerFeatureShape;
-const LanguagesImpl: new () => Languages = MonikerFeature(DiagnosticFeature(InlayHintFeature(InlineValueFeature(TypeHierarchyFeature(LinkedEditingRangeFeature(SemanticTokensFeature(CallHierarchyFeature(_LanguagesImpl)))))))) as (new () => Languages);
+export type Languages = _Languages & CallHierarchy & SemanticTokensFeatureShape & LinkedEditingRangeFeatureShape & TypeHierarchyFeatureShape & InlineValueFeatureShape & InlayHintFeatureShape & DiagnosticFeatureShape & MonikerFeatureShape & InlineCompletionFeatureShape;
+const LanguagesImpl: new () => Languages = InlineCompletionFeature(MonikerFeature(DiagnosticFeature(InlayHintFeature(InlineValueFeature(TypeHierarchyFeature(LinkedEditingRangeFeature(SemanticTokensFeature(CallHierarchyFeature(_LanguagesImpl))))))))) as (new () => Languages);
```

Comment:
Same as with the client. The InlineCompletionFeature first must come as a proposed feature users need to opt in in the beginning. Registering a propsed feature on the server is a little bit more tricky and I can help with this. An example can be found here: https://github.com/microsoft/vscode-languageserver-node/blob/9dd23aa6d06cada41964f972021924503929bfdd/server/src/common/api.ts

Reply:
I think I did this correctly but the need to cast `createConnection` in testServer.ts gave me some pause. Let me know if it should be adjusted further.